code/_lighting/SimpleLight.dm

```diff
@@ -249,12 +273,16 @@
 	show_mdir_light()
 
 	if (length(mdir_light_rgbas) == 1) //dont loop/average if list only contains 1 thing
-		for(var/obj/overlay/simple_light/medium/directional/mdir_light in src.mdir_lights)
-			if(mdir_light.dist == mdir_light_dists[mdir_light_dists.len])
-				mdir_light.color = rgb(rgba[1], rgba[2], rgba[3], min(255, rgba[4]))
+		var/alpha = rgba[4]
+		var/alpha_scaled = min(255, alpha / 2)
```

Comment:
0.5 != 0.4, is this just because it is cheaper to >>?

Reply:
oops was copypasting